.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: required
+language: go
+os:
+  - linux
+  - osx
+go_import_path: github.com/bazelbuild/sandboxfs
+go:
+  - 1.8
+  - 1.9
+env:
+  - SANDBOXFS=sandboxfs
+
+before_install:
+  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo apt-get update; fi
+  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo apt-get install -qq libfuse-dev pkg-config fuse user-mode-linux; sudo mknod /dev/fuse c 10 229; sudo chmod 666 /dev/fuse; fi
+  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew install Caskroom/cask/osxfuse; sudo /bin/cp -RfX /usr/local/opt/osxfuse/Library/Filesystems/osxfusefs.fs /Library/Filesystems; sudo chmod +s /Library/Filesystems/osxfusefs.fs/Support/load_osxfusefs; sudo mknod /dev/fuse c 10 229; sudo chmod 666 /dev/fuse; fi
```

Comment:
What do you think about having an `admin/travis-install.sh` script to contain all this logic and an `admin/travis-build.sh` script for the build logic below? Then it'd be easier to make sure they are correct, as one could run them directly.